test/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller_test.dart

```diff
@@ -483,4 +490,47 @@ void main() {
 
     tearDown(Get.deleteAll);
   });
+
+  group('spamMailboxId:test', () {
+    test('should returns spam mailbox ID if it exists', () {
+      // Arrange
+      final spamMailboxId = MailboxId(Id('spam-id'));
+      final junkMailboxId = MailboxId(Id('junk-id'));
+      final mapDefaultMailboxIdByRole = {
+        PresentationMailbox.roleSpam: spamMailboxId,
+        PresentationMailbox.roleJunk: junkMailboxId,
+      };
+      mailboxDashboardController.setMapDefaultMailboxIdByRole(mapDefaultMailboxIdByRole);
+      // Act
+      final spamId = mailboxDashboardController.spamMailboxId;
+
+      // Assert
+      expect(spamId, equals(spamMailboxId));
```

Comment:
why not prefer `junk`?

Reply:
We only need to take one because both have the same id. Only the role is different.

Reply:
But `junk` is in spec, I prefer to use `junk` if we have 2 roles in mailbox list

Reply:
Done